assets/js/modules/tagmanager/hooks/useExistingTagEffect.js

```diff
@@ -38,11 +38,9 @@ export default function useExistingTagEffect() {
 	const containerID = useSelect( ( select ) =>
 		select( MODULES_TAGMANAGER ).getContainerID()
 	);
-	const existingTagPermission = useSelect( ( select ) =>
-		select( MODULES_TAGMANAGER ).getTagPermission( existingTag )
-	);
-	const hasExistingTagPermission = useSelect( ( select ) =>
-		select( MODULES_TAGMANAGER ).hasExistingTagPermission()
+
+	const allContainers = useSelect( ( select ) =>
```

Comment:
As mentioned before, this selector isn't necessary. While the IB rightly calls for removing all permission related logic from this hook, the hook itself was dependent on the permissions data for the account ID to select. This hook was essentially slated for removal as part of https://github.com/google/site-kit-wp/issues/4713 but it may as well happen here as there is nothing it can or should do now that the permissions data has been removed.

Let's also remove this hook here rather than patching it up only to be removed later.